tools/osbuilder/rootfs-builder/agent.cfg

```diff
@@ -0,0 +1,47 @@
+aa_kbc_params = "offline_sev_kbc::100.64.0.203:44444"
```

Comment:
it might be best i the second parameter (after the `::`) was just `null` (as in the string `null` not actually null)